docs/managing-environments.md

```diff
@@ -129,4 +129,13 @@ poetry env remove 3.7
 poetry env remove test-O3eWbxRl-py3.7
 ```
 
+You can delete more than one environment at a time.
+```bash
+poetry env remove 3.6 3.7 3.8
```

Comment:
```suggestion
poetry env remove python3.6 python3.7 python3.8
```

This looks to be by executable and not by name.

Reply:
Sounds good if that is the best practice. I always do it the other way.  Good to know!